mail/templates/email_base.html

```diff
@@ -239,7 +239,7 @@
 	            <tr>
 	                <td style="padding: 20px;">
                       {% block logo %}
-                      <img src="{{ base_url }}/static/images/mit.png" width="160" height="80" alt="{{ site_name }}" style="height: auto; margin-bottom: -10px; margin-right: 10px;"/>
+                      <img src="{{ base_url }}/static/images/logo-mitx.png" width="160" height="80" alt="{{ site_name }}" style="height: auto; margin-bottom: -10px; margin-right: 10px;"/>
```

Comment:
Can you a more relevant name to the logo file that will specifically represent MITxOnline. e.g. maybe name it `mitx-online-logo`. Note that this would apply to the newly added logo file rename.

Reply:
updated file name